src/main/java/fr/communaywen/core/claim/ClaimListener.java

```diff
@@ -51,8 +55,12 @@ public void onBlockPlace(BlockPlaceEvent event) {
         UUID playerUuid = player.getUniqueId();
         for (RegionManager region : AywenCraftPlugin.getInstance().regions) {
             if (region.isInArea(event.getBlock().getLocation()) && !region.isTeamMember(playerUuid)) {
-                event.setCancelled(true);
-                player.sendMessage("§cCe n'est pas chez vous");
+                if(player.isOp()){
+                    //la personne est OP elle peut donc poser des blocs
+                }else{
+                    event.setCancelled(true);
+                    player.sendMessage("§cCe n'est pas chez vous");
+                }
```

Comment:
```suggestion
```suggestion
                if (!player.isOp()) {
                    event.setCancelled(true);
                    player.sendMessage("§cCe n'est pas chez vous");
                }
```
```